src/jit/codegencommon.cpp

```diff
@@ -714,6 +714,8 @@ regMaskTP Compiler::compNoGCHelperCallKillSet(CorInfoHelpFunc helper)
 #if defined(_TARGET_AMD64_)
             // this helper doesn't trash RSI and RDI
             return RBM_CALLEE_TRASH_NOGC & ~(RBM_RSI | RBM_RDI);
+#elif defined(_TARGET_ARM64_)
+            return RBM_R12 | RBM_R15;
```

Comment:
Based upon the x86 set it is likely correct to have R12 and R15 here.
This function is only called on x86/x64 in emitxarch.cpp